services/frontend/src/language/es.json

```diff
@@ -12,7 +12,7 @@
 
     "settingsTitle": "Configuración",
     "settingsLanguages": "Ingles/Español",
-    "settingsNotifications": "Notificación",
+    "settingsNotif  ications": "Notificación",
```

Comment:
Please remove the whitespace in `settingsNotifications`

Reply:
Removed and sent a commit to _localization branch in my fork. Please review and let me know if there´s anything else I´m missing, thank you.

![image](https://user-images.githubusercontent.com/49529903/64480788-81cf0c00-d18b-11e9-9793-c020169ce56f.png)

Reply:
@royzumbado I'm still seeing the space in the [file](https://github.com/royzumbado/eos-rate/blob/_localization/services/frontend/src/language/es.json), did you push your commit?

Reply:
@arturocr oh, I missed the push, I have pushed it now via GitHub desktop, could you please review/confirm?

Reply:
Looks good. 😃